charts/harvester-cloud-provider/102.0.0+up0.1.14/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Harvester Cloud Provider
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.26.0-0'
+  catalog.cattle.io/namespace: kube-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
+  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.8.0-0'
```

Comment:
This range looks "more correct" in previous released version of charts - https://github.com/rancher/charts/blob/425972dfa60be22c850bdfe77fcd2eda0fd5d08c/charts/harvester-cloud-provider/101.0.0%2Bup0.1.13/Chart.yaml#L8.
Any specific reason we're extending range to include 2.6.1+ here?

This seems to be due to removal of `*.patch` files and has other effects like `permits-os` annotation no longer including `windows`.

Is this intentional?

Reply:
@snasovich For the Harvester CSI and CCM chart, we are using exactly the same version for both Rancher v2.6 and v2.7 releases, and those charts are not used by the `Windows` so it should be fine to be removed. thanks.

Reply:
@guangbochen , I'm not _the_ expert on charts in Rancher, but I believe we generally want to maintain separate major versions for each Rancher minor versions (100.x.y for 2.6 line, 101.x.y for 2.7 line - and actually 102.x.y for 2.7.2+ to designate support for k8s 1.25 and handling of PSP removal).

So it will be best to retain `'>= 2.7.0-0 < 2.8.0-0'` range here. @aiyengar2 @PennyScissors , please chime in as _the_ charts experts.

Also, we're now missing `catalog.cattle.io/upstream-version` annotation that was previously added via `.patch` file.

Reply:
if that's preferred we should be able to patch those values accordingly via the `*.patch` files.  cc @Vicente-Cheng WDUT?

Reply:
Updated! Keep the `*.patch` files.
@snasovich, could you help to review it again? Thanks!

Reply:
@snasovich Yes, that's correct. The range for these charts should be `'>= 2.7.0-0 < 2.8.0-0'`. Additionally, the chart needs to be updated to support PSPs and k8s 1.25, and the version updated to `102.x.y` to indicate that. Assuming the chart supports k8s 1.25, the kube-version annotation is correct (`catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.26.0-0'`), but if these chart uses PSPs they need these changes https://github.com/rancher/rancher/issues/40331